cp3pt0-deployment/setup_singleton.sh

```diff
@@ -134,17 +146,17 @@ function install_licensing() {
         SOURCE_NS="ibm-licensing"
     fi
 
-    create_namespace ibm-licensing
+    create_namespace "${LICENSING_NAMESPACE}"
 
     target=$(cat <<EOF
         
   targetNamespaces:
-    - ibm-licensing
+    - ${LICENSING_NAMESPACE}
 EOF
 )
-    create_operator_group "ibm-licensing-operator-app" "ibm-licensing" "$target"
-    create_subscription "ibm-licensing-operator-app" "ibm-licensing" "$CHANNEL" "ibm-licensing-operator-app" "${LICENSING_SOURCE}" "${SOURCE_NS}" "${INSTALL_MODE}"
-    wait_for_operator "ibm-licensing" "ibm-licensing-operator"
+    create_operator_group "ibm-licensing-operator-app" "${LICENSING_NAMESPACE}" "$target"
+    create_subscription "ibm-licensing-operator-app" "${LICENSING_NAMESPACE}" "$CHANNEL" "ibm-licensing-operator-app" "${LICENSING_SOURCE}" "${SOURCE_NS}" "${INSTALL_MODE}"
```

Comment:
Would you mind helping to update the operator namespace and its package name to `ibm-licensing-operator`?

I noticed that in the standalone CatalogSource, the packageName is changed.

```
status:
  catalogSource: ibm-licensing-catalog
  catalogSourceDisplayName: IBMCS Licensing Test Operators
  catalogSourcePublisher: IBM
  catalogSourceNamespace: openshift-marketplace
  provider:
    name: IBM
  packageName: ibm-licensing-operator
```